avionics/Adafruit_BNO055.cpp

```diff
@@ -669,6 +736,32 @@ byte Adafruit_BNO055::read8(adafruit_bno055_reg_t reg )
   return value;
 }
 
+/**************************************************************************/
+/*!
+    @brief  Reads an 8 bit value over I2C
+*/
+/**************************************************************************/
+byte Adafruit_BNO055::read8v2(adafruit_bno055_reg2_t reg )
```

Comment:
search and find all instances of read8v2 into read8, write8v2 into write8
that's all you'd need for overloading